src/clauses/Union.ts

```diff
@@ -36,7 +37,12 @@ export class Union extends Clause {
     }
 
     public all(): this {
-        this.includeAll = true;
+        this.unionType = "ALL";
+        return this;
+    }
+
+    public distinct(): this {
+        this.unionType = "DISTINCT";
```

Comment:
LGTM, raise or not raise if this.unionType is already defined? I'm unopinionated here so I'm happy either way!